kubermatic-addons/addon-manifests/flux2.yaml

```diff
@@ -0,0 +1,231 @@
+apiVersion: kubermatic.k8s.io/v1
+kind: AddonConfig
+metadata:
+  name: flux2
+spec:
+  description: Open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
+  formSpec:
+      - displayName: Git repository URL
+        helpText: URL of the Git repository used as Flux source (only HTTP/S URLs are currently supported)
+        internalName: gitRepositoryUrl
+        required: true
+        type: text
+      - displayName: Git repository branch
+        helpText: Name of the Git branch to checkout and monitor for changes (defaults to "master")
```

Comment:
what about defaulting to "main" as most of git providers has done such switch on the new repositories